docs/data_management/lts/interfaces.md

```diff
@@ -38,13 +44,15 @@ conda activate s3
 pip install s3cmd
 ```
 
+Please note that the instructions mentioned above are specific to the Cheaha system. To transfer data between your personal computer and LTS, you will need to install `s3cmd` or `s5cmd` on your machine. Please refer to this [section](#installation-of-s3cmd-and-s5cmd-on-personal-system) for installation instructions specific to your operating system.
+
 <!-- markdownlint-disable MD046 -->
 !!! note
 
     We manually install pip into the conda environment so that `pip` will install `s3cmd` into the conda environment as opposed to `$HOME/.local`. This way, you do not need to add the `.local` folder to your path whenever you want to use `s3cmd`.
 <!-- markdownlint-enable MD046 -->
 
-### s3cmd
+### s3cmd Configuration
```

Comment:
Apologies if I wasn't clear before, or if this isn't complete. I'd like to have an `s3cmd` subsection and an `s5cmd` subsection for symmetry.

Right now `s5cmd` has an introductory paragraph, but `s3cmd` does not.

So we'd have the following. The subheadings "Configuring" and "Commands" would stay hidden due to the `toc_depth` we have.

- s3cmd
    - Configuring s3cmd
    - s3cmd Commands
- s5cmd
    - Configuring s5cmd
    - s5cmd Commands

Reply:
Thanks for clarifying again. I have incorporated the changes.